zubhub_frontend/zubhub/src/views/edit_profile/editProfileScripts.js

```diff
@@ -135,37 +135,26 @@ export const deleteAccount = (username_el, props, toast) => {
  *
  * @todo - describe function's signature
  */
-export const editProfile = (e, props, toast) => {
-  e.preventDefault();
-  props.setFieldTouched('username', true);
-  props.setFieldTouched('email', true);
-  props.setFieldTouched('phone', true);
-  props.setFieldTouched('password', true);
-  props.setFieldTouched('user_location', true);
+export const editProfile = (values, formikBag, toast) => {
```

Comment:
was the changes to this function because of the package upgrades?

Reply:
no, but you notice how I removed `props.setFieldTouched`. This function is connected to the `handleSubmit` function from `formik`. This make this submit function to only and only be called once validation is passed. This in comparison is a better way to handle data submission with formik. 
The formikBag contains all formik properties

Reply:
I have a suggestion @coderatomy , I think it'll help both of us if you avoid trying to make improvements and fix issues in our forms (this applies to everything else but specifically to forms).

Those forms are already terribly regressed (the previous person that worked on forms especially the project creation form does not care so much about making sure that the translations and validations work as they do before the changes, and I wasn't around to look more closely at things at the time so they got away with it).

If you avoid touching forms at all, then we can more confidently review this PR as it's own thing. But if you mess with forms, then we have to review the whole form functionality along the changes in this PR, and we might not be able to merge this PR on time before outreachy ends if we try fixing all these issues in the forms.

So I think for both of our sanity, we should avoid touching the forms. We should have a separate PR that will attempt to fix all the issues in the forms, and there are a lot.